cpp/ps0004/main.cpp

```diff
@@ -2,36 +2,33 @@
 using namespace std;
 
 int main() {
+    // prompting user for number 1.5/1.5
     float num;
     char ch;
 
     cout << "Enter a number between 32 and 126: ";
     cin >> num;
 
-
-    if (num < 32 || num > 126) {
-        cout << "Invalid: number is out of the range." << endl;
+    // checking invalidaty of user's input 4/4
+    if (num == 0) {
+        cout << "Invalid character/s or zero value." << endl;
         return 1;
     }
 
-
     int test = num;
     if (num != test) {
         cout << "invalid: a floating point number" << endl;
-        return 1;
+        return 2;
     }
-
-
-    if (num == 0) {
-        cout << "Invalid character/s or zero value." << endl;
-        return 1;
+    
+    if (num < 32 || num > 126) {
+        cout << "Invalid: number is out of the range." << endl;
+        return 3;
     }
 
-
     ch = num;
-
     cout << "The character for ASCII code " << num << " is: " << ch << endl;

```

Comment:
- [x] declared a variable to store the user input
- [x] chose a suitable data type for the variable
- [x] prompted user to enter a number
- [x] used `std::cin` in a right way to get user's input